src/Microsoft.Diagnostics.EventFlow.Inputs.Serilog/LoggerSinkConfigurationEventFlowExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Diagnostics.EventFlow;
+using Microsoft.Diagnostics.EventFlow.Inputs;
+using Serilog.Configuration;
+using Serilog.Debugging;
+using System;
+using System.Linq;
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extends <see cref="LoggerSinkConfiguration"/>
+    /// </summary>
+    public static class LoggerSinkConfigurationEventFlowExtensions
+    {
+        /// <summary>
+        /// Publish Serilog events through a <see cref="DiagnosticPipeline"/>.
+        /// </summary>
+        /// <param name="loggerSinkConfiguration">The <c>WriteTo</c> object exposed by <see cref="LoggerConfiguration"/>.</param>
+        /// <param name="diagnosticPipeline">A configured <see cref="DiagnosticPipeline"/>. At least one Serilog input
+        /// must be configured for the pipeline.</param>
+        /// <returns>A <see cref="LoggerConfiguration"/> allowing configuration to continue.</returns>
+        /// <remarks>The method will select the first <see cref="SerilogInput"/> in the pipeline. If a specific input is desired instead,
+        /// this can be passed to <see cref="LoggerSinkConfiguration.Sink(Core.ILogEventSink, Events.LogEventLevel, Core.LoggingLevelSwitch)"/>
+        /// instead.</remarks>
+        public static LoggerConfiguration EventFlow(this LoggerSinkConfiguration loggerSinkConfiguration, DiagnosticPipeline diagnosticPipeline)
+        {
+            if (loggerSinkConfiguration == null) throw new ArgumentNullException(nameof(loggerSinkConfiguration));
+            if (diagnosticPipeline == null) throw new ArgumentNullException(nameof(diagnosticPipeline));
+
+            var input = diagnosticPipeline.Inputs.OfType<SerilogInput>().FirstOrDefault();
+
+            if (input == null)
+            {
+                SelfLog.WriteLine("{0}: A Serilog input has not been added to the diagnostic pipeline; events will not be published to EventFlow", nameof(LoggerSinkConfigurationEventFlowExtensions));
+                return loggerSinkConfiguration.Sink(new LoggerConfiguration().CreateLogger()); // A "null" sink.
```

Comment:
I initially had an exception thrown here, but then considered that since the pipeline is configured using JSON, it's possible users might want to enable the Serilog input in some environment, and omit it in another. I was concerned that throwing an exception might introduce friction for this use case. Keen to fit with the rest of EventFlow so happy either way if you think an exception would be less surprising.